icepyx/core/read.py

```diff
@@ -54,6 +54,44 @@ def _make_np_datetime(df, keyword):
     return df
 
 
+def _get_track_type_str(grp_path) -> tuple[str, str]:
+    """
+    Determine whether the product contains ground tracks, paths, or profiles and
+    parse the string/label the dimension accordingly.
+
+    Parameters
+    ----------
+    grp_path : str
+        The group path for the ground track, path, or profile.
+
+    Returns
+    -------
+    track_str : str
+       The string for the ground track, path, or profile of this group
+    spot_dim_name : str
+        What the dimension should be named in the dataset
+    """
+
+    import re
+
+    # e.g. for ATL03, ATL06, etc.
+    if re.match(r"gt[1-3]['r','l']", grp_path):
+        track_str = re.match(r"gt[1-3]['r','l']", grp_path).group()
+        spot_dim_name = "spot"
+
+    # e.g. for ATL09
+    elif re.match(r"profile_[1-3]", grp_path):
+        track_str = re.match(r"profile_[1-3]", grp_path).group()
+        spot_dim_name = "profile"
+
+    # e.g. for ATL11
+    elif re.match(r"pt[1-3]", grp_path):
+        track_str = re.match(r"pt[1-3]", grp_path).group()
+        spot_dim_name = "path"
```

Comment:
Up to you whether you want to change this from 'path' to 'pair_track' (or just 'pair'). Would need to change some of the code below too :slightly_smiling_face:

```suggestion
        spot_dim_name = "path"
```